VAMobile/src/api/personalInformation/getPersonalInformation.tsx

```diff
@@ -40,6 +44,7 @@ export const usePersonalInformation = (options?: { enabled?: boolean }) => {
     queryKey: personalInformationKeys.personalInformation,
     queryFn: () => getPersonalInformation(),
     meta: {
+      analyticsUserProperty: UserAnalytics.vama_environment(ENVIRONMENT),
```

Comment:
Binny already added the environment event back (and in a better place than personal info, which was this weird leftover thing from MVP launch) - https://github.com/department-of-veterans-affairs/va-mobile-app/pull/8373

Reply:
Good catch, didn't even realize I added that here. Just removed this